### DISCUSSION TITLE
Passing OIDC User's Information to Destination Clusters

### DISCUSSION THREAD

**Discussion by ahmadalkhansa** [argoproj/argo-cd#18444] [General]
Hi,

I was wondering if it is possible to pass the information of authenticated users to destination clusters when performing actions on applications. The information would be username, groups, token issuer or user's access token itself. It would allow the use of dynamic admission controllers for fine-grained authorization and resource management. It can also be used to allow the authentication of users directly with the destination cluster in the case of passing access tokens to the API server of Kubernetes.

I can see that through Kubernetes events, user actions are traceable. There is also the possibility to check the application creation initiator. However, I cannot use validation controllers to manage fine-grained access control. 

Thank you,
Ahmad

**Comment by agaudreault** [comment]
This is not currently supported. You can follow https://github.com/argoproj/argo-cd/issues/7689 for more details

**Reply by ahmadalkhansa** [reply]
Thank you.
